docs/Lua API.rst

```diff
@@ -1073,6 +1088,15 @@ Announcements
   Uses the type to look up options from announcements.txt, and calls the above
   operations accordingly. The units are used to call ``addCombatReportAuto``.
 
+* ``dfhack.gui.autoDFAnnouncement(report,text[,log_failures])``
```

Comment:
could you link to where the `&` escape semantics are documented?

Reply:
Those are from reverse engineering the parsing function. I've added the info here now.